CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,73 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+
+# Format
+
+## Header name
+
+`Correlation-Context`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to [rfc](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2).
+
+## Field value
+
+`name1=value1[;type=(number|bool|string)], name2=value2[;type=(number|bool|string)]`
+
+**Limits:**
+1. Maximum number of name-value pairs: `180`.
```

Comment:
wait, why not 42? :-)

Reply:
lost in outdated comments...

proposed new limits. The idea of proposed limits:

url should fit easily as a value
overall size should not be driven by max number multiply by max size as it will become too big
max number of properties matches max number of cookies typically supported by browser for a single domain (generally browsers supports less, just took Chrome's number for a reference)

I'll add correlation-format_rationale.md as @adriancole suggests